app/controllers/application_controller.rb

```diff
@@ -20,6 +20,7 @@ def conversation
   protected
 
   def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
```

Comment:
Please explain why you are adding this line

Reply:
The ParameterSanitizer deals with permitting specific parameters values for each Devise scope in the application.

The sanitizer knows about Devise default parameters (like password and password_confirmation for the `RegistrationsController`), and you can extend or change the permitted parameters list on your controllers.